source/includes/rest/_subscribers.md

```diff
@@ -117,6 +119,14 @@ On January 7, 2019 we renamed Subscribers to People to better represent who you
       <td><code>phone</code></td>
       <td>The subscriber's primary phone number.</td>
     </tr>
+    <tr>
+      <td><code>sms_number</code></td>
+      <td>The subscriber's mobile phone number in E.164 formatting.</td>
+    </tr>
+    <tr>
+      <td><code>sms_consent</code></td>
+      <td>A boolean describing whether the person has granted consent to receive marketing and other communication via SMS.</td>
+    </tr>
```

Comment:
We should update these descriptions to match the others.

It's too bad we have to duplicate the descriptions in three spots. Wonder if there's a simple, low-effort way to resolve this.